03-transactions.md

```diff
@@ -827,6 +836,102 @@ at each bucket is a prefix of the desired index.
 
 # Appendix A: Expected Weights
 
+## Expected Weight of the Funding Transaction (v2 Channel Establishment)
+
+The *expected weight* of a funding transaction is calculated as follows:
+
+      inputs: 40 bytes + var_int + `scriptlen`
+        - previous_out_point: 36 bytes
+          - hash: 32 bytes
+          - index: 4 bytes
+        - var_int: ? bytes (dependent on `scriptlen`)
+        - script_sig: `scriptlen`
```

Comment:
line +216 of bolt 02 in this pr only allows witness previous outputs
the length of the scriptSig is thus 0

```suggestion
      inputs: 41 bytes
        - previous_out_point: 36 bytes
          - hash: 32 bytes
          - index: 4 bytes
        - var_int: 1 byte
        - script_sig: 0 bytes
```